src/test/resources/db/schema.sql

```diff
@@ -49,6 +55,9 @@
     alter table if exists document 
        add constraint UK_dte6vwi6c7hh3rcta5ace6k3t unique (document_data_id);
 
+    alter table if exists document_data 
+       add constraint UK_pk8e8r0wijpujsxn25gob65si unique (document_data_binary_id);
```

Comment:
Should constraint not have a proper name instead of a random one?

Reply:
Yes absolutely! But I just couldn't fix this one.

There is either a bug in JPA/HIbernate, or a weakness in my skills? I have really tried to fix this, but I just couldn't.

If anyone has an idea how to fix this I would greatly appreciate it! :-)

Reply:
Could not find a fix either.  Guess we are SOL on this for now.